src/KdbOptions.h

```diff
@@ -25,6 +52,15 @@ namespace Options
   // String options
   const std::string PARQUET_VERSION = "PARQUET_VERSION";
 
+  // Dict options
+  const std::string NULL_MAPPING = "NULL_MAPPING";
+
+  // Null mapping options
+  const std::string NM_INT_16 = "INT16";
+  const std::string NM_INT_32 = "INT32";
+  const std::string NM_STRING = "STRING";
+  const std::string NM_LARGE_STRING = "LARGE_STRING";

```

Comment:
Hmm, I think lowercase would be better for the null mapping option strings, e.g.:
```
const std::string NM_INT_16 = "int16";
```
I realise it doesn't match the top level options such as `PARQUET_VERSION` or `NULL_MAPPING` but would would be more consistent with the datatype constructor names in `arrowkdb.q`